docs/faq.md

```diff
@@ -32,6 +32,10 @@ GPUStack supports heterogeneous clusters spanning NVIDIA, AMD, Ascend, Hygon, Mo
 
 ## Installation
 
+!!! note
+
+    GPUStack v2 uses a unified image. For all hardware, use `gpustack/gpustack:latest`.
```

Comment:
1. No one checks here before installation. Move the note to installation docs. I think only legacy users will mess it up. Not sure if migration.md is a better place.

2. `GPUStack v2 uses a single unified container image for all GPU device types.`

Reply:
@gitlawr 

<img width="2436" height="1287" alt="image" src="https://github.com/user-attachments/assets/e478f333-53aa-48b3-8ec5-27abefa7a194" />